web/template/admin/admin_tabs/server-notifications.gohtml

```diff
@@ -1,51 +1,47 @@
 {{define "server-notification-edit"}}
-    <div class="dark:bg-secondary-lighter mt-4 bg-gray-100 rounded-md shadow-md min-w-min">
-        <form class="grid grid-cols-3 text-3 mb-2 p-5 gap-4"
-              action="/api/serverNotification/{{if .Model.ID}}{{.Model.ID}}{{else}}create{{end}}" method="post">
-            <input type="hidden" name="id" value="{{if .}}{{.Model.ID}}{{end}}">
-            <label class="col-start-1 col-span-3 uppercase">
-                <span class="text-4 uppercase text-sm font-semibold">Message</span>
-                <input type="text" name="text"{{if .}} value="{{.Text}}" {{end}}>
-            </label>
-            <label class="col-start-1 col-span-1 uppercase my-8 ">
-                <span class="text-4 uppercase text-sm font-semibold">From</span>
-                <input class="from" type="text" name="from" id="from{{if .}}{{.Model.ID}}{{end}}" placeholder="2021-07-06 12:00"{{if .}} value="{{.FormatFrom}}" {{end}}>
-            </label>
-            <label class="col-start-2 col-span-1 uppercase my-8">
-                <span class="text-4 uppercase text-sm font-semibold">Expires</span>
-                <input class="expires" type="text" name="expires" id="expires{{if .}}{{.Model.ID}}{{end}}" placeholder="2021-07-06 14:00"{{if .}} value="{{.FormatExpires}}" {{end}}>
-            </label>
-            <div class="col-start-3 col-span-1 uppercase ml-3 my-8">
-                <span class="text-4 text-sm font-semibold">Type</span>
-                <div>
-                    <input {{if .}}{{if not .Warn}}checked{{end}}{{else}}checked{{end}} class="w-auto" type="radio"
-                           id="info" name="type" value="info">
-                    <label class="capitalize" for="info">Info</label>
-                </div>
-                <div>
-                    <input {{if .}}{{if .Warn}}checked{{end}}{{end}} class="w-auto" type="radio" id="warning"
-                           name="type" value="warning">
-                    <label class="capitalize" for="warning">Warning</label>
-                </div>
-            </div>
-        </form>
-        <div class="flex m-auto justify-center">
-            <button class="bg-gray-500 shadow w-full text-white rounded-sm hover:bg-gray-600 dark:hover:bg-gray-600 rounded-sm dark:bg-gray-900 px-2 py-2 ml-6 mr-6 mb-6 font-semibold" title="Submit notification">
-                Submit notification
-            </button>
+<form class="dark:bg-secondary p-6 bg-gray-100 rounded-lg shadow-md grid grid-cols-3 gap-4"
+    action="/api/serverNotification/{{if .Model.ID}}{{.Model.ID}}{{else}}create{{end}}" method="post">
+    <h2 class="text-1 font-bold col-span-full">Create a new notification</h2>
+    <input type="hidden" name="id" value="{{if .}}{{.Model.ID}}{{end}}">
+    <label>
+        <span class="text-2 font-bold">Message</span>
+        <input type="text" name="text" {{if .}} value="{{.Text}}" {{end}}>
```

Comment:
If you like, you could take a look at the new input styling of the login page ([login.gohtml](https://github.com/joschahenningsen/TUM-Live/blob/dev/web/template/login.gohtml), also: [components.md](https://github.com/joschahenningsen/TUM-Live/blob/dev/docs/styling/components.md)) and apply that styling to some of the inputs of the admin tabs for which the new design looks better than the old one.

Reply:
I've just pushed an update to the create-lecture-hall-ui if you want to take a look at the new design theme: https://github.com/joschahenningsen/TUM-Live/pull/674

Reply:
Ok, gonna use the new design theme and merging your PR into this branch.

Reply:
Do you want to keep the 4/5 width on the container? Seems a bit wasteful.

Reply:
the 4/5 is an arbitrary design decision by me. Therefore, absolutely not necessary. Do as you think fits the UI best!